lib/ts/recipe/emailpassword/index.ts

```diff
@@ -166,7 +167,8 @@ export default class Wrapper {
         });
     }
 
-    static SignInAndUp = (prop?: any) => EmailPassword.getInstanceOrThrow().getFeatureComponent("signinup", prop);
+    static SignInAndUp = (prop: PropsWithChildren<{ redirectOnSessionExists?: boolean; userContext?: any }> = {}) =>
+        EmailPassword.getInstanceOrThrow().getFeatureComponent("signinup", prop);
```

Comment:
why does this PR have this change? Wasn't this change already merged into the base branch?